LearningJournal/scripts/LJAPI.py

```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
```

Comment:
why do you have the same code in your repository twice?  Kill one of these two files.